metadata/validator/__init__.py

```diff
@@ -0,0 +1,80 @@
+from datetime import date
+from pydantic import BaseModel
+
+
+class client_data(BaseModel):
```

Comment:
Careful about the name of class and variable

Not all the class and variable name of this file are align with the Python convention

convention for class name: CamelCase
convention for variable: snake_case


```suggestion
class ClientData(BaseModel):
```

Be careful to look back at this document as many class and variable need to be rename